ansible_collections/f5networks/f5_modules/plugins/modules/bigip_gtm_wide_ip.py

```diff
@@ -387,7 +387,7 @@ class ModuleParameters(Parameters):
     @property
     def last_resort_pool(self):
         if self._values['last_resort_pool'] in [None, '', 'none']:
-            return ''
+            return None
```

Comment:
should be 'none' not None as it won't send anything to the device ? what is the fix here ? Last resort pool API accepts 'none' as parameter to effectively remove last resort pool setting, or am i missing something?

Reply:
The iControl REST API no longer accepts an empty 'lastResortPool' property for wide IP objects. 

We are using None in the rest of the code, so i left it as None to make consistency and it fixed the issue. I tried "none" and it has same result, if you want change it to "none" instead of None

Reply:
have you checked on earlier versions of bigip you must not break other versions with this

Reply:
Tested in bigip v13.0, v14.1.3,v15.0. along with v14.1.4